lab5.h

```diff
@@ -0,0 +1,204 @@
+﻿#include <iostream>
+#include <string>
+using namespace std;
+class Candidate {
```

Comment:
I have no idea was is candidat, you should chenge class name, for example to "nominee" or some other name

Reply:
this is comdition of task, i have name this class so